io.github.andrewmkiss.Demeter.yml

```diff
@@ -0,0 +1,190 @@
+app-id: io.github.andrewmkiss.Demeter
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: dathena
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=host
+
+modules:
+  - name: hdf4_lib
+    disabled: false 
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - 'hdf4_lib/install.sh'
+    sources:
+      - type: archive
+        url: https://support.hdfgroup.org/ftp/HDF/releases/HDF4.2.16/src/CMake-hdf-4.2.16.tar.gz
+        sha256: e8115564daa913c274f88558c2bc5d6685ef51531c80bf95f6dd92ced1f7b64f
+      - commands:
+        - set -e
+        - cd CMake-hdf-4.2.16/
+        - sed -i "79s/set.*/set \( INSTALLDIR \"\/app\")/" HDF4config.cmake
+        - ctest -S HDF4config.cmake,BUILD_GENERATOR=Unix -C Release -V -O hdf4.log
+        - cd build
+        - make install
+        - make clean
+        dest: hdf4_lib
+        dest-filename: install.sh
+        type: script
+
+  - name: gnuplot
+    disabled: false
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=/app/ --without-qt
+      - make
+      - make install
+      - make clean
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/gnuplot/files/gnuplot/6.0.0/gnuplot-6.0.0.tar.gz
+        sha256: 635a28f0993f6ab0d1179e072ad39b8139d07f51237f841d93c6c2ff4b1758ec
+
+  - name: ncurses
+    disabled: false
+    buildsystem: autotools
+    no-autogen: true
+    config-opts: --prefix=/usr
```

Comment:
```suggestion
```

1. syntax is invalid (it's an array)
2. `/usr` as prefix is not possible. I'm pretty sure build would fail with that.

Reply:
or flaptak-builder to did print a warning, and ignored it like it should.